src/content/docs/de/guides/imports.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Statische Inhalte
+description: Erfahre, wie du verschiedene Inhalte in Astro importierst.
```

Comment:
I think “Inhalte” is OK here but wanted to throw out “Dateitypen” or “Dateiformaten” as alternatives to think about. The English is “content types”.